platform/processor/live-instrument/src/test/kotlin/integration/CompositeDatatypesLiveBreakpointTest.kt

```diff
@@ -0,0 +1,169 @@
+package integration
+
+import io.vertx.core.json.Json
+import io.vertx.core.json.JsonObject
+import io.vertx.junit5.VertxTestContext
+import org.junit.jupiter.api.Assertions.assertEquals
+import org.junit.jupiter.api.Assertions.assertTrue
+import org.junit.jupiter.api.Test
+import spp.protocol.SourceServices.Provide.toLiveInstrumentSubscriberAddress
+import spp.protocol.instrument.LiveBreakpoint
+import spp.protocol.instrument.LiveSourceLocation
+import spp.protocol.instrument.event.LiveInstrumentEvent
+import spp.protocol.instrument.event.LiveInstrumentEventType
+import spp.protocol.marshall.ProtocolMarshaller
+import java.math.BigInteger
+import java.time.*
+import java.util.*
+import java.util.concurrent.TimeUnit
+
+@Suppress("UNUSED_VARIABLE")
+open class CompositeDatatypesLiveBreakpointTest : LiveInstrumentIntegrationTest() {
+
+    private fun compositeDatatypes() {
+        val activeSpan = startEntrySpan("compositeDatatypes")
+        val date = Date(1L)
+        val duration = Duration.ofSeconds(1)
+        val instant = Instant.ofEpochSecond(1)
+        val localDate = LocalDate.now()
+        val localTime = LocalTime.of(1, 2, 3, 4)
+//        val offsetDateTime = OffsetDateTime.now()
+//        val offsetTime = OffsetTime.now()
+//        val zonedDateTime = ZonedDateTime.now()
+//        val zoneOffset = ZoneOffset.ofTotalSeconds(1)
+////        val zoneRegion = ZoneRegion("1",  ZoneRules.of(zoneOffset))
+        val bigInteger = BigInteger.TEN
+//        val clazz = Class.forName("Ezpeepee")
```

Comment:
This class doesn't exist. You should be creating a new class in this file and finding that.